projects/storefrontstyles/scss/components/layout/_skip-link.scss

```diff
@@ -0,0 +1,27 @@
+%cx-skip-link {
```

Comment:
I'm going to propose an alternative CSS here, shouldn't impact any of the other code, so your good to go with making changes